### ISSUE TITLE
Missing ellipsis for long calendar event names

### ISSUE THREAD

**Issue by FelixMalfait** [twentyhq/twenty#5267]
I just noticed we don't handle long event names properly:

<img width="615" alt="Screenshot 2024-05-03 at 13 49 23" src="https://github.com/twentyhq/twenty/assets/6399865/bda25e28-f345-47cf-af71-7a0125f3dd60">

**Comment by orinamio** [comment]
I'll look into this issue. Can you assign me to it?

**Comment by FelixMalfait** [comment]
Sure! Thanks again @orinamio

**Comment by charlesBochet** [comment]
I have closed https://github.com/twentyhq/twenty/pull/5360 today but the issue still remains!
There is a technical strategy in https://github.com/twentyhq/twenty/pull/5360 comments :)

**Comment by charlesBochet** [comment]
Hi @Nil2000, these are the correct instruction, you'll indeed need WSL and a powerful enough setup on Windows to setup the project

**Comment by Nil2000** [comment]
Thanks @charlesBochet for the info I have set up correctly. But now I have another issue I don't see any Calender events to check this. Can you help me produce this event? 
I am thinking of sending a Google Meet Request from one of my Gmail accounts to tim@apple.dev. Do you think this will work?

**Comment by Faisal-imtiyaz123** [comment]
@lucasbordeau Can you assign me this please.

**Comment by mittalsam98** [comment]
@Bonapara pls assign this to me, I will look into this

**Comment by lucasbordeau** [comment]
@Faisal-imtiyaz123 Assigned

**Comment by Nabhag8848** [comment]
Related, Same with Email, Files, Tasks, Notes. 

<img width="1752" alt="Files" src="https://github.com/user-attachments/assets/9de1036a-f3b5-41b0-a49c-eec190b7d21c">


https://github.com/user-attachments/assets/beb3cd35-24a2-440c-b1cd-0b2c30600da5




<img width="1644" alt="Notes" src="https://github.com/user-attachments/assets/ed89020a-0d54-4510-925c-ae6e5f10aef9">

<img width="1644" alt="Task" src="https://github.com/user-attachments/assets/d38f6ef8-d9d9-4e79-987b-0789b7b475cb">

**Comment by Nabhag8848** [comment]
@FelixMalfait working on it.

**Comment by Faisal-imtiyaz123** [comment]
@Nabhag8848 This issue is assigned to me. Kindly once check the assignee

**Comment by FelixMalfait** [comment]
@Faisal-imtiyaz123 because it was assigned a month ago I think @Nabhag8848 assumed you weren't working on it anymore which is fair because ; see  https://github.com/twentyhq/twenty/blob/main/.github/CONTRIBUTING.md#getting-started 

Thanks again to both of you for your contributions! More than enough work for everyone haha :)

**Comment by Nabhag8848** [comment]
Thanks @FelixMalfait for mentioning that ! 

@Faisal-imtiyaz123 you might be interested in checking out this: https://appwrite.io/blog/post/make-open-source-healthier or a even a talk by same speaker https://www.youtube.com/watch?v=YuUt5qABv_k  I just went to an event yesterday which talked about what we have gone through rn :) "Contributors role on maintaining Community Health".

**Comment by Faisal-imtiyaz123** [comment]
@Nabhag8848 Thanks for sharing it. Will surely see it.

**Comment by rajeshkrishnait** [comment]
Is this still open? Can i work on it?

**Comment by FelixMalfait** [comment]
Probably @Nabhag8848 got stuck! So yes feel free to raise a PR! Thank you

(just make sure to look at the previously closed PR)

**Comment by Bonapara** [comment]
/oss.gg 150

**Comment by Bonapara** [comment]
@Abhithube01 would you be able to open a PR today on this one?

**Comment by sonu4435** [comment]
Hello @Bonapara  ,

I've just been assigned to this issue, but I wanted to check whether it’s still open or if it has already been resolved, as there have been a lot of discussions, and the issue appears to still be open. If it's still active, could you kindly direct me to any relevant instructions or resources I need to be aware of?

Thanks in advance!

**Comment by Bonapara** [comment]
Yes, still a thing @sonu4435, thanks! 

https://github.com/user-attachments/assets/2cd39cc9-792f-4347-a034-166f5c4c4e1b

**Comment by sonu4435** [comment]
Hi Bob,

I’ve been assigned to this issue, and from what I’ve seen, the problem seems to be limited to the calendar section. I haven’t noticed any similar issues in other areas.

However, I don’t have a calendar currently linked to my account, and I suspect the calendar is fetched via Google OAuth. I’m receiving a 520 error, which is preventing me from logging in and seeing the calendar’s behavior. As a result, I’m unable to fully address the issue at the moment.

Could you clarify how I can find that or guide me on how to proceed? I can clone the repo and try to troubleshoot, but I might encounter the same issue unless it’s resolved first.
![image](https://github.com/user-attachments/assets/62589ec0-0c31-4257-a731-fb9c6fc0ecf7)
![image](https://github.com/user-attachments/assets/32e0bbce-e469-4ab2-9767-1f0f5fb8f8b5)

**Comment by Bonapara** [comment]
I don't think so, but we should do it when tackling https://github.com/twentyhq/twenty/issues/6793

**Comment by balvind3r** [comment]
<img width="786" alt="image" src="https://github.com/user-attachments/assets/920278a0-b002-4b8b-a89e-31880ef2bcc1">
I am having trouble keeping the frontend up...

**Comment by asadath1395** [comment]
I would like to work on this one

**Comment by Bonapara** [comment]
Sure @asadath1395, thanks for contributing!

**Comment by tinchil** [comment]
Hi @Bonapara , Would it be possible to assign myself to this issue? I would like to work on this task

**Comment by FelixMalfait** [comment]
Thanks @tinchil I think there's already an ongoing PR from @asadath1395 that @bosiraphael will review

**Comment by lucasbordeau** [comment]
@bosiraphael Is this still relevant ?

**Comment by FelixMalfait** [comment]
Probably yes https://github.com/twentyhq/twenty/pull/8528

**Comment by bosiraphael** [comment]
Yes

**Comment by Bonapara** [comment]
Sure @Lucifer4255, thanks for contributing!

**Comment by Lucifer4255** [comment]
![Image](https://github.com/user-attachments/assets/b61e0ed6-66c3-4b92-9849-b80911709ea9)
Is this ok @Bonapara ?

**Comment by Bonapara** [comment]
Hi @Lucifer4255 no, the user chips should still stick to the right.

The issue is there: When the text is too long it should be cropped and not push the user chip outside.

![Image](https://github.com/user-attachments/assets/0bb08c31-4584-4f53-aa06-cf00ba8f022e)

**Comment by Lucifer4255** [comment]
![Image](https://github.com/user-attachments/assets/8ba4de69-7eff-4f3e-b108-52346d769ae7)
I Think this might be it 

I am raising a PR for this

**Comment by Bonapara** [comment]
Thanks @Lucifer4255
